tdrs-frontend/src/components/UploadReport/UploadReport.jsx

```diff
@@ -97,14 +97,19 @@ function UploadReport({ handleCancel, header }) {
     )
 
     Promise.all(uploadRequests)
-      .then(() => {
+      .then((responses) => {
         setLocalAlertState({
           active: true,
           type: 'success',
           message: `Successfully submitted section(s): ${formattedSections} on ${new Date().toDateString()}`,
         })
         clearErrorState()
-        logger.alert(`Submitted data files`)
+
+        // Create LogEntries in Django for each created ReportFile
+        logger.alert(`Submitted data file(s)`, {
+          files: responses.map((response) => response?.data?.id),
+          activity: 'upload',
+        })
```

Comment:
Use `responses`, which is a list of the newly submitted ReportFile objects created in Django, and use their IDs to create LogEntries.

Reply:
Makes sense. If we could add the additional data to write to the Django Logs as well, that would be ideal

Reply:
What additional data do you mean?

Reply:
> What additional data do you mean?

The same things we added to the output in Django Admin. File name, type, etc.

Reply:
Confused as to what you mean, these are already being added?

Reply:
Gotcha, thanks for clarifying.